vs-code/vscode.md

```diff
@@ -91,18 +101,13 @@ Briefly summarize your findings so you can share with the class.
 
 ### Check for Understanding
 
-Apprentices make a cheat sheet (could be on paper or in a note on their Desktop) that reminds them how to **use shortcuts** to:
+Form small groups and discuss:
 
-- Open VS Code
-- Open a specific folder in VS Code
-- Open a specific file in VS Code
-- Find and replace multiple occurrences of one word in the same file
-- Click on multiple characters in one file at the same time
-- Comment and uncomment text
-- Indent and dedent lines of text
-- Move one line up or down
-- Open a file side by side
-- Open integrated terminal
+1. How do you open a specific file or folder in VS Code?
```

Comment:
actually it's better to use list in 1-numbered style

```
1. One
1. Two
1. Three
```

it gives

1. One
1. Two
1. Three

and in case of editing no need to renumber items

Reply:
I changed these to bullets for consistency but changed other ordered lists in the file to 1-numbered style.